src/systems/systems/src/landing_gear/mod.rs

```diff
@@ -744,6 +756,95 @@ impl LandingGearControlInterfaceUnit {
     fn status(&self) -> LgciuStatus {
         self.status
     }
+
+    fn discrete_word_1(&self) -> Arinc429Word<u32> {
```

Comment:
Would you mind making these accessible to other systems (like the FWC in future?). You could either make the method public, or add it to a public trait and implement that (e.g. like `systems::shared::LgciuWeightOnWheels`);

Reply:
I'll just make it public, I don't see a use of making it a trait, since it's not going to be different on different aircraft, or am I missing something?